playwright_test/tests/richMessage_Card_Carousel.spec.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * richMessage_Card_Carousel.spec.js
+ * playwright_test/test
+ * Created by Archit on 20/01/2023. 
+ */
+
+import { test, expect } from '@playwright/test';
+import { WIDGET_LOCATORS, LOCATORS, COMMON_VALUES } from '../locaterPackage/kmLocators.js';
+import { URL , APP_ID } from '../utils/kmSecret';
+import { SCRIPT } from '../utils/kmScript';
+
+let page;
+
+// Launching widget
+  test.beforeAll(async ({browser}) => {
+    page = await browser.newPage();
```

Comment:
is this same launch code be which is present in other test cases as well? If yes, move it to a common code file and call it from all places

Reply:
ya this is the same but when we add more test cases in the same file we need this to be in the same file because we can shift some of the test code to beforeAll()

Reply:
may be move common code to a separate file, and then call that function from everywhere

Reply:
if we can take this in last because have to do a lot of trials to separate beforeAll() and afterAll() with this getting lots of errors due to the playwright version and other stuff I will take this in last as code formatting ?? if you are ok with this

Reply:
ok